enterprise/frontend/src/metabase-enterprise/sandboxes/confirmations.ts

```diff
@@ -25,7 +25,7 @@ export function getSandboxedTableWarningModal(
     )
   ) {
     return {
-      title: t`Change access to this database to “Sandboxed”?`,
+      title: t`Change access to this database to “Row and column security“?`,
```

Comment:
```suggestion
      title: t`Change access to this database to “Row and column security”?`,
```

The closing quote is backwards